packages/extensions-sdk/src/cli/run.ts

```diff
@@ -25,7 +25,8 @@ program
 	.option('-l, --language <language>', 'overwrite the language to use')
 	.option('-f, --force', 'ignore the package manifest')
 	.option('-w, --watch', 'watch and rebuild on changes')
-	.option('-m --sourceMaps', 'include source maps in output')
+	.option('--no-minify', 'disable minification')
+	.option('--sourcemap', 'include source maps in output')
```

Comment:
Lets add the same `-m` flag shorthand here 👍🏻

Reply:
I intentionally removed the shorthand because I would expect a `-m` flag to stand for Minification instead of sourceMap generation 🤔 Also, compared to the other options, it feels lika a less important option.

Reply:
Fair! I figured it could be a way to alleviate the breaking change